vic/drivers/classic/src/read_vegparam.c

```diff
@@ -371,28 +372,28 @@ read_vegparam(FILE  *vegparam,
     }
 
     // Determine if we have bare soil
-    if (temp[0].Cv_sum > 1.0) {
-        log_warn("Cv exceeds 1.0 at grid cell %d, fractions being "
-                 "adjusted to equal 1", gridcel);
+    if (Cv_sum > 1.0) {
```

Comment:
It *could* be an opportunity to do that. The context for this change was that `Cv_sum` was actually never used outside `read_vegparam` so it was removed from `veg_con_struct`.  In this particular case, it doesn' hurt anything to normalize `Cv_sum` in the case it is just slightly `> 0.` so I don't think it is necessary here.